hub/core/query/query.py

```diff
@@ -47,11 +51,14 @@ def execute(self) -> List[int]:
                     for tensor in self._tensors
                 }
                 p.update(self._groups)
-                if eval(self._cquery, p):
-                    idx_map.append(local_idx)
-                    self._pg_callback(local_idx, True)
-                else:
-                    self._pg_callback(local_idx, False)
+                try:
+                    if eval(self._cquery, p):
+                        idx_map.append(local_idx)
+                        self._pg_callback(local_idx, True)
+                    else:
+                        self._pg_callback(local_idx, False)
+                except Exception:
```

Comment:
Same as above

Reply:
As we should honor all the exceptions, I am not sure what kind of exceptions can be there